src/rtcSessionHelpers.ts

```diff
@@ -128,13 +127,13 @@ const widgetPostHangupProcedure = async (
   // we need to wait until the callEnded event is tracked on posthog.
   // Otherwise the iFrame gets killed before the callEnded event got tracked.
   await new Promise((resolve) => window.setTimeout(resolve, 10)); // 10ms
-  widget.api.setAlwaysOnScreen(false);
+  await widget.api.setAlwaysOnScreen(false);
```

Comment:
I think we need both. We always want to send the hangup action but also need to await the setAlwaysOnScreen.

I opted for try catch because the formatter otherwise puts everything in its own line:
```
  await widget.api
    .setAlwaysOnScreen(false)
    .catch((e) =>
      logger.error("Failed to set call widget `alwaysOnScreen` to false", e),
    );
    ```